quickref/qsharp-quick-reference.tex

```diff
@@ -198,15 +211,22 @@
     Return a value     & \texttt{return \emph{value}} \\
     Stop with an error & \texttt{fail "\emph{Error message}"} \\
   \end{keysref}
-
+   
+   \begin{keysref}{Conjugations}
+	ApplyWith & \texttt{operation \emph{Name}(\emph{in0} : \emph{type0}, ...) : \emph{returnType} \{} \newline
```

Comment:
I would call this entry `apply ... within`, since `ApplyWith` is a name of the function we used to have for this functionality, not functionality itself, and limit it to the actual `within ... apply` entries.

I would also provide a bit more context on what it does (i.e., that it is equivalent to withinBlock - applyBlock - Adjoint withinBlock sequence)